packages/vision/src/helpers.js

```diff
@@ -74,11 +74,12 @@ var coerceImage = (image, callback) => {
  *   asking for the single feature annotation.
  */
 var _createSingleFeatureMethod = featureValue => {
-  return function(image, options) {
-    return this.annotateImage({
+  return function(image, options = {}, callOptions = undefined) {
```

Comment:
> JavaScript idioms have always preferred a configuration object over multiple arguments.

Sure. What is not clear in this case is how to set up that object.

My problem with trying to combine the request object with what I renamed `callOptions` is that this would be a single one-off against the entire rest of the library. A user could reasonably ask, "Why is _this_ one different?"

Honestly, maybe the right alternative is to just drop `callOptions` _completely_ for these methods. It is rarely used, and it would be reasonable to say that if you need it, you have to call `annotateImage`.